src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -137,10 +194,36 @@ protected override void BeginProcessing()
         protected override void ProcessRecord()
         {
             bool result = true;
+            string resolvedPath = string.Empty;
+            string jsonToParse = string.Empty;
+
+            if (Json != null)
+            {
+                jsonToParse = Json;
+            }
+
+            if (Path != null)
```

Comment:
I guess this should be a `else if`, given that they belong to different parameter sets.

Reply:
Fixed.